app/controllers/wizards_controller.rb

```diff
@@ -17,12 +17,13 @@ def new
 
   def create
     if current_user
-      @wizard = Wizard.new(wizards_params.merge({ creator_id: current_user.id }))
+      @wizard = Wizard.new(wizards_params)
+      @wizard.creator_id = current_user.id
```

Comment:
you could remove the id here - I find it to be more expressive

`@wizard.creator = current-user`